### PR TITLE
Nodeenv support

### PR DESCRIPTION
**Pull request by rmad17**
Added function for [nodeenv](https://github.com/ekalinin/nodeenv) in prompt.

P.S. I looked into some previous closed PRs and assumed that `next` is the branch for the next release hence opening PR to `next`.

### CODE REVIEW

**Comment by bhilburn** [comment]
@rmad17 - Thanks for the PR! And yes, this is the correct branch =)

Can you add one more commit to this PR to update the README to add the new segment to the segment list & provide a brief description?

File: powerlevel9k.zsh-theme [hunk 1]
```diff
@@ -557,6 +557,15 @@ prompt_nvm() {
   $1_prompt_segment "$0" "$2" "green" "011" "${node_version:1}" 'NODE_ICON'
 }
 
+# NodeEnv Prompt
+prompt_nodeenv() {
+  local nodeenv_path="$NODE_VIRTUAL_ENV"
+  if [[ -n "$nodeenv_path" && "$NODE_VIRTUAL_ENV_DISABLE_PROMPT" != true ]]; then
```
Comment (bhilburn) [inline powerlevel9k.zsh-theme]: Is `$NODE_VIRTUAL_ENV_DISABLE_PROMPT` something common to `nodeenv` or is it specific to this Powerlevel9k prompt segment?

Comment (rmad17) [inline powerlevel9k.zsh-theme]: Its related to [nodeenv](https://github.com/ekalinin/nodeenv/blob/2cd7de2005c4e33cc324ecd2cf84c995ebcd2d19/nodeenv.py#L929
). 
I will do the necessary update to README shortly.

**Comment by bhilburn** [comment]
@rmad17 - Sounds great!! Thanks so much, and I look forward to getting this upstreamed =)

**Comment by rmad17** [comment]
@bhilburn Its a great feeling to give back to the community. I have updated the README. Let me know if I need to make any further changes.

**Comment by bhilburn** [comment]
@rmad17 - Solid PR! Merged!

Thanks so much for the great contribution!

**Comment by rmad17** [comment]
@bhilburn Thanks :)

**Pull request merged**